run/image-processing/requirements.txt

```
@@ -1,6 +1,6 @@
 Flask==3.0.3
 google-cloud-storage==2.12.0
 google-cloud-vision==3.8.1
-gunicorn==22.0.0
+gunicorn==23.0.0
```

https://github.com/GoogleCloudPlatform/python-docs-samples/pull/13067#issuecomment-2621639283

```suggestion
gunicorn==22.0.0
```